setup.py

```diff
@@ -1,28 +1,25 @@
 import os
 from distutils.core import setup
-
+from watson import __version__
```

Comment:
is not good import the package in `setup.py`, because the package is not yet installed, then this can cause some problems when you are installing the package. In a django package (not specifically, this package), the problem of doing this, is if the package in require that django is already installed previously, but it is not already installed, will result in a ImportError (or other error) ...

my current solution to this problem, is create the function `get_version_and_author`, and use that to indirectly get version and author:

if `watson/__init__.py` contains

```python
"""
Multi-table search application for Django, using native database search engines.
Developed by Dave Hall.
<http://www.etianen.com/>
"""

__version__ = '1.2.3'
__author__ = 'Dave Hall'

```

the `setup.py` would look like:

```python
import io
import os
import re
from distutils.core import setup


def get_version_and_author(*file_paths):

    filename = os.path.join(os.path.dirname(__file__), *file_paths)
    with io.open(filename, encoding='utf8') as the_file:
        file_content = the_file.read()
        version_match = re.search(r"^__version__ = ['\"]([^'\"]*)['\"]",
                                  file_content, re.M)
        author_match = re.search(r"^__author__ = ['\"]([^'\"]*)['\"]",
                                 file_content, re.M)
        if version_match and author_match:
            return version_match.group(1), author_match.group(1)
    raise RuntimeError('Unable to find version and/or author string.')


VERSION, AUTHOR = get_version_and_author('watson', '__init__.py')

setup(
    name="django-watson",
    version=VERSION,
    description="Full-text multi-table search application for Django. Easy to install and use, with good performance.",
    long_description=open(os.path.join(os.path.dirname(__file__), "README.markdown")).read(),
    author=AUTHOR,
    author_email="dave@etianen.com",
    url="http://github.com/etianen/django-watson",
    zip_safe=False,
    packages=[
        "watson",
        "watson.management",
        "watson.management.commands",
        "watson.migrations",
        "watson.south_migrations",
        "watson.templatetags",
    ],
    package_data={
        "watson": [
            "locale/*/LC_MESSAGES/django.*",
            "templates/watson/*.html",
            "templates/watson/includes/*.html",
        ],
    },
    classifiers=[
        "Development Status :: 5 - Production/Stable",
        "Environment :: Web Environment",
        "Intended Audience :: Developers",
        "License :: OSI Approved :: BSD License",
        "Operating System :: OS Independent",
        "Programming Language :: Python",
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3.3',
        'Programming Language :: Python :: 3.4',
        'Programming Language :: Python :: 3.5',
        'Programming Language :: Python :: 3.6',
        "Framework :: Django",
    ],
)

```